digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -53,9 +53,12 @@
 
     public static TDOperator fromConfig(SystemDefaultConfig systemDefaultConfig, Map<String, String> env, Config config, SecretProvider secrets)
     {
-        String database = secrets.getSecretOptional("database").or(config.get("database", String.class)).trim();
-        if (database.isEmpty()) {
-            throw new ConfigException("Parameter 'database' is empty");
+        String database = null;
+        if (!(config.has("_type") && config.get("_type", String.class).equals("td_result_export"))) {
+            database = secrets.getSecretOptional("database").or(config.get("database", String.class)).trim();
+            if (database.isEmpty()) {
```

Comment:
If possible, can we avoid td_result_export specific behavior in common class?
For example,
```Java
public static TDOperator fromConfig(SystemDefaultConfig systemDefaultConfig, Map<String, String> env, Config config, SecretProvider secrets)
{
    return fromConfig(....  , true);  // To keep compatibility
}
public static TDOperator fromConfig(SystemDefaultConfig systemDefaultConfig, Map<String, String> env, Config config, SecretProvider secrets, boolean databaseRequired)
{
}
```
This is just only my idea.